adobe/brackets
# JavaScript quick edit performance tests

- Remove unused `onInlineGesture` arg in EditorManager._createEditorForDocument
- Refactor InlineTextEditor.close() to InlineWidget.close()
- Remove `browser` jslint option in MultiRangeInlineEditor
- Refactor JavaScript Quick Editor for performance testing
  -- Add performance tests using jQuery UI as test media
- Moved test/lib to test/thirdparty
- Change PerformanceReporter to show the measurement name
- Change PerformanceReporter to only output non-empty results

This pull request isn't as bad as it looks. The ugliest changes are just refactoring to expose methods for performance testing.

Also see https://github.com/adobe/brackets-scenario/pull/3 for test media
